CMakeLists.txt

```diff
@@ -171,6 +171,15 @@ if (CMAKE_COMPILER_IS_GNUCC OR CMAKE_COMPILER_IS_CLANGCC OR CMAKE_COMPILER_IS_IC
         list(APPEND OSD_COMPILER_FLAGS -Wall -Wextra)
     endif()
 
+    if(WIN32)
```

Comment:
These defines are done for MSVC below, but also needs to be applied for non-MSVC builds.